src/train.jl

```diff
@@ -73,9 +73,15 @@ function sciml_train(loss, _θ, opt, _data = DEFAULT_DATA;
     data = _data
   end
 
+  progress && @logmsg(LogLevel(-1),
+  "Training",
+  _id = :DiffEqFlux,
+  message="init params = "*string(ps),

```

Comment:
no need to print this. Instead have the message be the current loss?

Reply:
Hmm, I didn't see the message at all.

Reply:
if you scroll over the progress bar it should show.